### DISCUSSION TITLE
How to use this package ?

### DISCUSSION THREAD

**Discussion by sagar2514** [yume-chan/ya-webadb#690] [General]
I am trying to implement a demo for controlling android tablets through web browser. I am creating simple HTML project. How do we use this project?

**Comment by yume-chan** [comment]
You can check out the documentation at https://docs.tangoapp.dev/

**Reply by sagar2514** [reply]
on mac i am getting "Failed to execute 'transferIn' on 'USBDevice': A transfer error has occurred. Ensure that the USB port is not in use (i.e. adb server is running)." 

but windows it works fine
